flake8_quotes/__init__.py

```diff
@@ -12,31 +12,53 @@ class QuoteChecker(object):
     QUOTES = {
         # When user wants only single quotes
         '\'': {
-            'good_single': '\'',
-            'good_multiline': '\'\'\'',
-            'bad_single': '"',
-            'bad_multiline': '"""',
+            'good': '\'',
+            'bad': '"',
         },
+
+        # When user wants only double quotes
         '"': {
-            'good_single': '"',
-            'good_multiline': '"""',
-            'bad_single': '\'',
-            'bad_multiline': '\'\'\'',
+            'good': '"',
+            'bad': '\'',
         },
     }
 
+    MULTILINE_QUOTES = {
```

Comment:
I'm positive we discussed this at length on this PR previously; we want to keep them as dicts to reduce the amount of noise in the PR. If we want to change implementation of existing components, then that should be discussed/done in another issue/PR.